kernels/tf_recommenders.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# To build the kernel:  ./kernels/reinforcement_learning.sh
+# To remove the kernel: ./kernels/reinforcement_learning.sh remove
+#
+# This scripts will create a ipython kernel named $MODULE
+# populated with the reqs in ./notebooks/$MODULE/requirements.txt
+
+
+MODULE=recommendation_systems
+ENVNAME=tf_recommenders_kernel
+REPO_ROOT_DIR="$(dirname $(cd $(dirname $BASH_SOURCE) && pwd))"
+
+# Cleaning up the kernel and exiting if first arg is 'clean'
```

Comment:
nit: the comment here says first arg is "clean" but the code shows "remove"

Reply:
I forgot to update this. Thanks!